Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -22,6 +22,7 @@ class MessageDTO: NSManagedObject {
     @NSManaged var showReplyInChannel: Bool
     @NSManaged var replyCount: Int32
     @NSManaged var extraData: Data?
+    @NSManaged var isBounced: Bool
```

Comment:
@adolfogarza does storing `isBounced` separately from local state allow us to revert `syncingFailed` state for messages marked locally as bounced when we open the channel? 🤔

Reply:
that is correct.